From 43b059d9a1ed7d5adb1e561942861865069c2897 Mon Sep 17 00:00:00 2001
From: Andrei Botila <andrei.botila@nxp.com>
Date: Mon, 23 Aug 2021 12:27:08 +0300
Subject: [PATCH 05/17] s32: reorder dtb files in the dts makefile

commit 43b059d9a1ed7d5adb1e561942861865069c2897 from
https://source.codeaurora.org/external/autobsps32/linux

The dtb's should be ordered alphabetically in the freescale dts Makefile.

Upstream-Status: Pending

Issue: ALB-7471
Signed-off-by: Andrei Botila <andrei.botila@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/Makefile | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/Makefile b/arch/arm64/boot/dts/freescale/Makefile
index ef4194d1faa3..a6d8fd6ceac4 100644
--- a/arch/arm64/boot/dts/freescale/Makefile
+++ b/arch/arm64/boot/dts/freescale/Makefile
@@ -51,24 +51,24 @@ dtb-$(CONFIG_ARCH_MXC) += imx8qxp-ai_ml.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8qxp-colibri-eval-v3.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8qxp-mek.dtb
 
-dtb-$(CONFIG_ARCH_S32) += fsl-s32v234-bbmini.dtb
-dtb-$(CONFIG_ARCH_S32) += fsl-s32v234-campp-primary.dtb
-dtb-$(CONFIG_ARCH_S32) += fsl-s32v234-campp-secondary.dtb
-dtb-$(CONFIG_ARCH_S32) += fsl-s32v234-evb28899.dtb
-dtb-$(CONFIG_ARCH_S32) += fsl-s32v234-evb.dtb
-dtb-$(CONFIG_ARCH_S32) += fsl-s32v234-evbbcm.dtb
-dtb-$(CONFIG_ARCH_S32) += fsl-s32v234-hpcsom.dtb
-dtb-$(CONFIG_ARCH_S32) += fsl-s32v234-pciebcm.dtb
-dtb-$(CONFIG_ARCH_S32) += fsl-s32v234-pcie.dtb
+dtb-$(CONFIG_ARCH_S32) += fsl-s32g274a-simulator.dtb
 dtb-$(CONFIG_ARCH_S32) += fsl-s32g274a-bluebox3.dtb
 dtb-$(CONFIG_ARCH_S32) += fsl-s32g274a-emu.dtb
 dtb-$(CONFIG_ARCH_S32) += fsl-s32g274a-evb.dtb
 dtb-$(CONFIG_ARCH_S32) += fsl-s32g274a-rdb.dtb
 dtb-$(CONFIG_ARCH_S32) += fsl-s32g274a-rdb2.dtb
-dtb-$(CONFIG_ARCH_S32) += fsl-s32g274a-simulator.dtb
 dtb-$(CONFIG_ARCH_S32) += fsl-s32g398a-emu.dtb
-dtb-$(CONFIG_ARCH_S32) += fsl-s32r45-simulator.dtb
 dtb-$(CONFIG_ARCH_S32) += fsl-s32r45-emu.dtb
 dtb-$(CONFIG_ARCH_S32) += fsl-s32r45-evb.dtb
-dtb-$(CONFIG_ARCH_S32) += fsl-s32v234-ccpb.dtb
+dtb-$(CONFIG_ARCH_S32) += fsl-s32r45-simulator.dtb
 dtb-$(CONFIG_ARCH_S32) += fsl-s32v234sbc.dtb
+dtb-$(CONFIG_ARCH_S32) += fsl-s32v234-bbmini.dtb
+dtb-$(CONFIG_ARCH_S32) += fsl-s32v234-campp-primary.dtb
+dtb-$(CONFIG_ARCH_S32) += fsl-s32v234-campp-secondary.dtb
+dtb-$(CONFIG_ARCH_S32) += fsl-s32v234-ccpb.dtb
+dtb-$(CONFIG_ARCH_S32) += fsl-s32v234-evb28899.dtb
+dtb-$(CONFIG_ARCH_S32) += fsl-s32v234-evb.dtb
+dtb-$(CONFIG_ARCH_S32) += fsl-s32v234-evbbcm.dtb
+dtb-$(CONFIG_ARCH_S32) += fsl-s32v234-hpcsom.dtb
+dtb-$(CONFIG_ARCH_S32) += fsl-s32v234-pciebcm.dtb
+dtb-$(CONFIG_ARCH_S32) += fsl-s32v234-pcie.dtb
-- 
2.17.1

